main.cpp

```diff
@@ -23,7 +23,7 @@ std::string createFileName();
 
 int main(int argc, char* argv[]) {
     if (argc != 2){
-	std::cerr << "Usage: submit filepath" << std::endl;
+	std::cerr << "Usage: " << BASH_NAME << " filepath" << std::endl;
```

Comment:
Where is the Variable/ Constant BASH_NAME specified?

Reply:
BASH_NAME is passed via g++ at compile time via the -D option. This ensures that the program uses The name the user chose while installing.
Please look at instal.sh for more details.